### PR TITLE
Armand van der Zwan - Feed API Challenge

### PR DESCRIPTION
**Pull request by armandzwan**

### CODE REVIEW

File: FeedAPIChallenge/Feed API/RemoteFeedLoader.swift [hunk 1]
```diff
@@ -18,7 +19,40 @@ public final class RemoteFeedLoader: FeedLoader {
...
+        let image_loc: String?
+        let image_url: URL
+    }
+    
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-        
+        client.get(from: url) { [weak self] (error) in
+            guard let self = self else { return }
+            switch error {
+            case .success((let data, let reesponse)):
```
Comment (mapostolakis) [inline FeedAPIChallenge/Feed API/RemoteFeedLoader.swift:37]: There's an extra "e" in the response name.

**Review by mapostolakis — COMMENTED** [review]
Hi @armandzwan,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs. I only left a comment regarding a small typo.

Please, feel free to fix and push a new commit.

Great job! ✅

File: FeedAPIChallenge/Feed API/RemoteFeedLoader.swift (hunk 1 above)
Comment (armandzwan) [reply FeedAPIChallenge/Feed API/RemoteFeedLoader.swift]: Thanks. I fixed it in the original commit.

https://github.com/essentialdevelopercom/ios-lead-essentials-feed-api-challenge/pull/59/commits/8f90da4e7a4411089e533c6a519b65563f042e33

Comment (mapostolakis) [reply FeedAPIChallenge/Feed API/RemoteFeedLoader.swift]: Awesome 👍

**Pull request closed without merging**